Sources/arm/ui/TabMaterials.hx

```diff
@@ -176,14 +176,16 @@ class TabMaterials {
 					}
 					if (ui.isHovered) {
 						ui.tooltipImage(imgFull);
-						ui.tooltip(Project.materials[i].canvas.name);
+						if (i < 9 && ui.isHovered) ui.tooltip(Project.materials[i].canvas.name + " - (" + Config.keymap.select_material + " " + (i+1) + ")");
```

Comment:
Looks like no need for `&& ui.isHovered` here.

Reply:
good catch